product_docs/docs/pgd/5/quick_start_aws.mdx

```diff
@@ -131,60 +131,61 @@ tpaexec configure democluster \
   --no-git \
   --hostnames-unsorted
 ```
-We included options to specify using AWS, a single location, and three data nodes running [EDB Postgres Advanced Server](/epas/latest/). By default, PGD also configures two [PGD Proxy](routing/proxy/) nodes and a [Barman](backup#physical-backup) node for backup. Rocky Linux is the default image for all nodes.
+These options specify to use AWS, a single location, and three data nodes running [EDB Postgres Advanced Server](/epas/latest/). By default, PGD also configures two [PGD proxy](routing/proxy/) nodes and a [Barman](backup#physical-backup) node for backup. Rocky Linux is the default image for all nodes.
 
 !!! Note Deployment platforms
-    Numerous other Linux platforms are supported as deployment targets for PGD - view [the EDB Postgres Distributed compatibility table](https://www.enterprisedb.com/resources/platform-compatibility) for details.
+    Other Linux platforms are supported as deployment targets for PGD. See the [EDB Postgres Distributed compatibility table](https://www.enterprisedb.com/resources/platform-compatibility) for details.
     
-    Observe that we don't have to deploy PGD to the same platform we're using to run TPA!
+    You don't have to deploy PGD to the same platform you're using to run TPA.
 
-We set the notional location of the nodes to `eu-west-1` using `--location-names`. We then activate the PGD proxies in that location using `--active-locations` set to the same location.
+The notional location of the nodes is set to `eu-west-1` using `--location-names`. Activate the PGD proxies in that location using `--active-locations` set to the same location.
```

Comment:
I think this is wrong. The --location-names is not notional when done on AWS as it deploys to that region. @josh-heyer thoughts?

Reply:
No. In fact, I should update this example to include `--regions eu-west-1` so as not to imply that location-names controls regions.

Reply:
Ok, then we leave as is.